ios-push-app.xcodeproj/xcuserdata/nostanle.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Typically we do not put any of the `xcuserdata` files in source control. Can you remove them?